starknet_py/net/account/account_client.py

```diff
@@ -171,52 +184,128 @@ async def get_balance(
 
         return (high << 128) + low
 
-    async def _prepare_execute_transaction(self, tx: InvokeFunction) -> Transaction:
+    async def prepare_invoke_function(
```

Comment:
`InvokeFunction` is a terrible name considering what is really happening. Can we brainstorm a better name and use it as alias for `InvokeFunction`?

Reply:
We decided that we'll discuss it later because we couldn't find a solution